testsuite/classlibrary/TestSCDoc.sc

```diff
@@ -0,0 +1,25 @@
+TestSCDoc : UnitTest {
+
+    test_helpSourceDirs_excludedExtensions {
+        var result;
+        var extensionPaths = [thisProcess.platform.systemExtensionDir, thisProcess.platform.userExtensionDir];
+        var prevSetting = LanguageConfig.excludeDefaultPaths;
+        LanguageConfig.excludeDefaultPaths = true;
+        SCDoc.initClass;
+        result = SCDoc.helpSourceDirs.any{|p| extensionPaths.any{|ep| ("^"++ep).matchRegexp(p)}};
```

Comment:
per our [style guide](https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#whitespace), this should be:

```supercollider
result = SCDoc.helpSourceDirs.any { |p| extensionPaths.any { |ep| ("^" ++ ep).matchRegexp(p) } };
```

normally i'm not a stickler for these things but there's a couple broken rules here ^^